ROMFS/px4fmu_common/init.d-posix/px4-rc.simulator

```diff
@@ -40,11 +40,19 @@ elif [ "$PX4_SIMULATOR" = "gz" ]; then
 		echo "INFO  [init] PX4_GZ_VERBOSE not explicitly set, defaulting to: $gz_verbose."
 	fi
 
-	gz_world=$( ign topic -l | grep -m 1 -e "/world/.*/clock" | sed 's/\/world\///g; s/\/clock//g' )
-	
-	gz_version_major=$( ign gazebo --versions | sed 's/\..*//g' )
-	gz_version_minor=$( ign gazebo --versions | sed 's/'"${gz_version_major}"\.'//; s/\..*//g' )
-	gz_version_point=$( ign gazebo --versions | sed 's/'"${gz_version_major}"\.'//; s/'"${gz_version_minor}"\.'//')
+	gz_command="gz"
+	gz_sub_command="sim"
+	if [ -z $(command -v $gz_command) ];
+	then
+			gz_command="ign"
+			gz_sub_command="gazebo"
+	fi
```

Comment:
Since we are checking for whether gazebo version is available, would it be possible to also throw an error if there is also no `ign` command available?